ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/ruleeditorhook.js

```diff
@@ -35,6 +35,9 @@
             showAlert();
         } else {
             let ruleEditorUri = '/aem/af/expeditor.html' + getFormContainerPath(editable) + "?fieldPath=" + editable.path + "&fieldId=" + getFieldId(editable);
+            if(window.location.pathname.startsWith('/lc/editor.html')) {
```

Comment:
we already have the context path in form container in its data attribute `data-cmp-context-path`. you need to find the closes form container from this editable and then read its data attribute.